Doc/library/contextlib.rst

```diff
@@ -327,10 +327,12 @@ Functions and classes provided:
 .. function:: redirect_stdout(new_target)
 
    Context manager for temporarily redirecting :data:`sys.stdout` to
-   another file or file-like object.
+   another :term:`file object`.
 
    This tool adds flexibility to existing functions or classes whose output
-   is hardwired to stdout.
+   is hardwired to :data:`sys.stdout`. This does not modify underlying file
+   objects or file descriptors. It sets the global :data:`sys.stdout` to the
+   provided value and at context exit sets it to the previous value.
```

Comment:
This should be left out entirely. That is already covered by the docs for `sys.stdout` and `sys.__stdout__`